### PR TITLE
Add synchronized option

### PR DESCRIPTION
**Pull request by minRa**

### CODE REVIEW

**Comment by stwiname** [comment]
Did these changes come from the repo we forked?

File: RNKeychainManager/RNKeychainManager.m [hunk 1]
```diff
@@ -255,12 +255,13 @@ - (void)insertKeychainEntry:(NSDictionary *)attributes withOptions:(NSDictionary
...
 }
 
-- (OSStatus)deletePasswordsForService:(NSString *)service
+- (OSStatus)deletePasswordsForService:(NSString *)service 
 {
   NSDictionary *query = @{
     (__bridge NSString *)kSecClass: (__bridge id)(kSecClassGenericPassword),
     (__bridge NSString *)kSecAttrService: service,
     (__bridge NSString *)kSecReturnAttributes: (__bridge id)kCFBooleanTrue,
+    (__bridge NSString *)kSecAttrSynchronizable: (__bridge id)kCFBooleanTrue
```
Comment (stwiname) [inline RNKeychainManager/RNKeychainManager.m]: Does this implicitly make everything synchronised?

**Comment by minRa** [comment]
> Did these changes come from the repo we forked?

No, it wasn't, I refered to react-native-keychain issue => https://github.com/oblador/react-native-keychain/issues/156#issuecomment-510165288  and https://developer.apple.com/documentation/security/ksecattrsynchronizable

File: RNKeychainManager/RNKeychainManager.m (hunk 1 above)
Comment (minRa) [reply RNKeychainManager/RNKeychainManager.m]: It seems like that but I might need a test to make sure it is

**Comment by developer-darpan** [comment]
code looks good to me 👍

but, if A & B are synced, and A logs out, then what will happen to B? will it be usable or we need to clear other account-related(non-secure) data from B too?

**Comment by minRa** [comment]
> code looks good to me 👍
> 
> but, if A & B are synced, and A logs out, then what will happen to B? will it be usable or we need to clear other account-related(non-secure) data from B too?

I will test this case once the build has done

**Pull request closed without merging**